arm-resources/resources/2016-02-01/swagger/resources.json

```diff
@@ -1877,6 +1877,22 @@
         "properties": {
           "type": "object",
           "description": "Gets or sets the resource properties."
+        },
+        "kind": {
```

Comment:
And using a model-as-string (inclusive set of values) type enum doesn't make sense?